app/uk/gov/hmrc/claimvatenrolmentfrontend/config/AppConfig.scala

```diff
@@ -50,6 +50,8 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   lazy val btaUrl: String = servicesConfig.getString("bta.url")
```

Comment:
It looks like on VSU in the SignUpComplete view we use this btaUrl not the relative one added